libcxx/include/locale

```diff
@@ -366,9 +366,11 @@ _LIBCPP_HIDE_FROM_ABI _ForwardIterator __scan_keyword(
 
 struct _LIBCPP_EXPORTED_FROM_ABI __num_get_base {
   static const int __num_get_buf_sz = 40;
+  static const size_t __int_chr_cnt = 26; // count of leading characters in __src used for parsing integers
+  static const size_t __fp_chr_cnt  = 28; // count of leading characters in __src used for parsing floating-point values
```

Comment:
I would move these down, directly below `static const char __src[33]; // "0123456789abcdefABCDEFxX+-pPiInN"`
```suggestion
  static const size_t __int_chr_cnt = 26; // count of leading characters in __src used for parsing integers ("012..X+-")
  static const size_t __fp_chr_cnt  = 28; // count of leading characters in __src used for parsing floating-point values  ("012..-pP")
```
I think adding a range makes it easier to understand which characters are in the range.

Reply:
Done. (Although the current formatting for `__fp_chr_cnt` is a bit ugly due to the long comment...)

Reply:
I agree, maybe move the comment before the declaration instead of a trailing comment.